src/main/resources/javamoney.properties

```diff
@@ -1,15 +1,23 @@
-# ECB Rates
+# Standard provider chain
+{1}conversion.default-chain=IDENT,ECB
+
+# Decimal rounding per provider
+{1}ecb.digit.fraction=6
+
 {1}load.ECBCurrentRateProvider.type=SCHEDULED
 {1}load.ECBCurrentRateProvider.period=03:00:00
-{1}load.ECBCurrentRateProvider.resource=/java-money/defaults/ECB/eurofxref-daily.xml
+{1}load.ECBCurrentRateProvider.resource=org/javamoney/moneta/convert/ecb/defaults/eurofxref-daily.xml
 {1}load.ECBCurrentRateProvider.urls=https://www.ecb.europa.eu/stats/eurofxref/eurofxref-daily.xml
 {1}load.ECBCurrentRateProvider.startRemote=false
 
-{1}load.ECBHistoric90RateProvider.type=LAZY
-{1}load.ECBHistoricRateProvider.type=LAZY
-{1}load.ECBHistoric90RateProvider.urls=
-{1}load.ECBHistoricRateProvider.urls=
+# Disable ECBHistoric90RateProvider
```

Comment:
Please add more comments with documenting changes in this file, because it is not very trivial to understand how chain of providers is configured.